.github/workflows/benchmarks.yml

```diff
@@ -99,7 +99,11 @@ jobs:
 
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.8"
+          python-version: 3.11
+
+      - name: Remove Dependencies versions
```

Comment:
It is not needed anymore.

Reply:
Yeah.... I think since I open this PR before you merged #6 , I think I have to merge this branch with main to avoid conflicts... And push the commit to this PR
Or I can simply resolve conflicts from here

Reply:
> I think I have to merge this branch with main to avoid conflicts... And push the commit to this PR

Yes, that is a good option

> Or I can simply resolve conflicts from here

Pressing "resolve conflicts" button is also a good option. (I think it will do exactly the same as the manual merge).

Reply:
This change is still something that is not needed anymore

Reply:
Hello @marcin-krystianc 
I removed the step!

You can review